Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
nominally, undoing a change implemented with an era, should be another era,
but we are told to not use it and now discuss how to turn things off instead with a cmsDriver command.

If I understand correctly, changes made by HCAL eras in this PR will not work on the old 2016 HCAL geometry.